index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Tribute Page</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@1,900&family=Yanone+Kaffeesatz:wght@200;300&display=swap" rel="stylesheet">
+    <style>
+        * {
+            margin: 0px auto;
+            padding: 0px;
+        }
+
+        #img-div {
+            background-color: rgb(227, 231, 255);
+        }
+
+        #image {
+            display: block;
+            max-width: 50%;
+            height: auto;
+            margin: 0px auto;
+            /* padding: 15px 15px; */
+            background-color: rgb(193, 219, 253);
+        }
+
+        #title {
+            margin: 0 auto;
+            text-align: center;
+            font-family: 'Lato', sans-serif;
+            background-color: #92bef0;
+            color: #061832;
+            position: sticky;
+            top: 1px;
+        }
+
+        blockquote {
+            text-align: center;
+            font-family: cursive;
+            background-color: #92bef0;
+            padding: 12px 15px;
+        }
+
+        #img-caption {
+            text-align: center;
+            font-family: cursive;
+            font-style: italic;
+
+        }
+
+        #tribute-info {
+            background-color: rgb(235, 235, 235);
+
+        }
+
+        h3 {
+            text-align: center;
+            /* margin: 0px 10px; */
+            padding: 5px 5px;
+            font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+            font-size: 24px;
+        }
+
+        p {
+            margin-left: 25px;
+            margin-right: 25px;
+            font-size: large;
+            font-family: 'Yanone Kaffeesatz', sans-serif;
+            font-weight: 900;
+            line-height: 150%;
+        }
+        p#footer{
+            padding: 25px 25px;
+            text-align: center;
+            font-family: cursive;
+            background-color: rgb(238, 238, 238);
+            font-style: oblique;
+            font-weight: 700;
+
+        }
+        a{
+            text-align: center;
+            text-decoration: none;
+        }
+        a:hover{
+            background-color: cornflowerblue;
+            border: 1px solid cornflowerblue;
+            border-radius: 10px;
+            color: crimson;
+        }
+        a:visited{
+            color: darkgrey;
+        }
+        
+        #created{
+            font-family: fantasy;
+            font-style: normal;
+            font-size: 20px;
+            text-align: center;
+        }
+    </style>

```

Comment:
My suggestion would be to use a style.css file to style your HTML Document as it keeps things organized and professional 😁